src/components/Main/Sakura.scss

```diff
@@ -0,0 +1,54 @@
+.sakura {
+    position:absolute;
+    display: flex;
+    justify-content: space-evenly;
+    top:-100px;
+    width:100%;
+    height: calc(100% + 100px);

```

Comment:
I would rather save 100px into some css variable, because top and height are connected here, e.g.
```
top: -@shiftUp;
...
height: calc(100% + @shiftUp);
```